features/show/tabs.feature

```diff
@@ -23,5 +24,8 @@ Feature: Show - Tabs
       end
     """
     Then I should see two tabs "Overview" and "Details"
-    And I should see "tab 1"
-    And I should see "tab 2"
+    And I should see the element "#overview span"
+    And I should not see the element "#details span"
+    Then I follow "Details"
+    And I should not see the element "#overview span"
+    And I should see the element "#details span"
```

Comment:
Also found this one: https://github.com/activeadmin/activeadmin/blob/master/features/step_definitions/tab_steps.rb#L1-L3, which could be useful here.